src/integrationTest/java/integration/tests/SystemEngineDatabaseMetaDataTest.java

```diff
@@ -147,7 +147,7 @@ void getTables(String catalog, String schemaPattern, String tableNamePattern, St
             ",%schema,%in%,table_name,false",
             ",%form%,%in%,table_schema,false",
             "{database},,%in%,table_catalog,false",
-            "{database},information_schema,%in%,account_id,false",
+            "{database},information_schema,%ac%,account_id,false",
             "{database},information%,%in%,name,false",
```

Comment:
I'm not sure I understand, what does this do?

Reply:
This exactly what I mentioned. Once upon a time this test found table named `running_queries` that is now not accessible from system engine. However pattern `%ac%` matches 2 tables: `accounts` and `rbac_roles`. I will modify this test and add expected tables instead of `emptyList=true|false`